### ISSUE TITLE
[FSE] Enable keyboard shortcut for saving site editor changes

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#29642]
## What problem does this address?

As part of the second round of testing for the FSE Outreach Program, [the following feedback was given](https://make.wordpress.org/test/2021/02/18/fse-program-testing-call-2-build-a-homepage-with-site-editing-blocks/#comment-1398):

> When editing, I expect CMD/CTRL + S to save my work. This works in a post/page editing experience. On OS X + Chrome, this prompts me to save the webpage.

Similar to this issue around saving autosaving https://github.com/WordPress/gutenberg/issues/29577, this is likely a bit complex due to the multi-entity saving process but I imagine is something users are used to having in place. 

## What is your proposed solution?

Perhaps using this shortcut pops open the saving process for example rather than saving everything.

**Comment by carolinan** [comment]
The need to add this missing short cut was also brought up during user testing on April 9.
